### DISCUSSION TITLE
Change Admin User Email

### DISCUSSION THREAD

**Discussion by AwesomeStickz** [WeblateOrg/weblate#8596] [Q&A] [answered]
How to change the admin user email? I tried updating the email in `docker-compose.overrirde.yml` but I can't login with that email but only with the username, is there any proper steps to do?

**Comment by nijel** [comment, marked as answer]
The admin user is not re-created in the usual setup, see https://docs.weblate.org/en/latest/admin/install/docker.html#envvar-WEBLATE_ADMIN_PASSWORD.

Just login as admin and change e-mail in a same way you would do with a normal user in the profile.

**Reply by AwesomeStickz** [reply]
Didn't see that option, thanks a lot!
